Client/Card.ts

```diff
@@ -47,7 +47,9 @@ export class Card extends http.Client<gracely.Error> {
 	static create(connection: string): { card: Card } {
 		return {
 			card: new Card(connection, {
-				public: connection.includes("https://api.pax2pay.com") ? "production-public-key" : "test-public-key",
+				public: connection.includes("https://api.pax2pay.com")
+					? "production-public-key"
```

Comment:
is it possible for this to be going to .app?

Reply:
fairly certain this will only be using .com